bam_markdup.c

```diff
@@ -265,7 +289,7 @@ static int64_t calc_score(bam1_t *b)
    read is leftmost of the pair. */
 
 
-static int make_pair_key(md_param_t *param, key_data_t *key, bam1_t *bam, long *warnings) {
+static int make_pair_key(md_param_t *param, key_data_t *key, bam1_t *bam, uint8_t rg_num, long *warnings) {
```

Comment:
Why is this `uint8_t` and not `int`, as used elsewhere?  (Also in `make_single_key()`.)

Reply:
I don't remember.  I think it was part of an earlier development phase.  I changed how it worked part way through.  I'll change it.